woocommerce/woocommerce-android
# [Shipping Labels Revamp] Introduce Printing networking layer and file creation support

Summary
==========
Fix issue #12392 and partially fix issue #12288 by introducing the networking layer and all File operations to display the Shipping Label for printing. 

This PR also connects the Printing action with the Purchased screen, but it's only to make it easy to test the Printing code. It doesn't implement anything regarding loading and error feedback. UI feedback and improvements will be added in the next PR for issue #12288. 

Screen Capture
==========
https://github.com/user-attachments/assets/0afcd884-c628-451e-a796-23b0270e638c

How to Test
==========
1. Open the main Shipping Label screen
2. Since the Purchased screen is only accessible from the landscape screen binding, switch the device to landscape mode
3. Hit the Purchase button
4. Once in the Purchased Screen, hit the Print Shipping Label and verify that for each of the Paper size, the Shipping Label is opened in a separate screen following the selected paper size.

Update release notes:


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
